<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QMainWindow" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1839</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1801</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>页</string>
     </attribute>
     <widget class="QGraphicsView" name="gvMain">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>520</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </property>
      <property name="interactive">
       <bool>true</bool>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_vector">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>721</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_test">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>440</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>流程测试</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_test_pos_y">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_test_action">
           <item>
            <property name="text">
             <string>NULL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_Light_Judge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_Rough_Detect</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_Rough_MoveOver</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_Fine_Detect</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_Fine_MoveOver</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_ReDetect</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Action_DetectOver</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>坐标Y：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>动作：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_test">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>坐标X：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_test_pos_x">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>440</y>
        <width>141</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>流程控制</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_run_process">
           <property name="text">
            <string>开始流程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop_process">
           <property name="text">
            <string>流程暂停</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_quit_process">
           <property name="text">
            <string>流程结束</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>440</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>结果显示</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_max_value"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_position_1"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_redet_value"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>坐标(触发前)：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>最大光通量：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>复检光通量：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_position_2"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>坐标(触发后)：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>复检延时：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_position_3">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>440</y>
        <width>233</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>数据处理</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,3">
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_maxValue"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>输入查找值：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>查找类型：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_search_value"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_search_result"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_search_value">
           <item>
            <property name="text">
             <string>坐标x</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>坐标y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>差值x</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>差值y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>光通量</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>结果所在列：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_search_value">
           <property name="text">
            <string>查找</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButton_search_max_value">
           <property name="text">
            <string>查找最大值</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>最大值结果：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>331</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>板卡运行日志</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>1270</x>
        <y>20</y>
        <width>501</width>
        <height>601</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <column>
       <property name="text">
        <string>最大值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>复检值</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>角度1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>角度2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>复检差值</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1839</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
